src/app/login/login-form.directive.ts

```diff
@@ -0,0 +1,12 @@
+import { Directive, ElementRef } from '@angular/core';
```

Comment:
Can we make this directive as a service or any global file , so that we can reuse this .